pkg/controller/template/template_controller.go

```diff
@@ -171,12 +182,54 @@ func (ctrl *Controller) deleteSecret(obj interface{}) {
 	}
 }
 
+func (ctrl *Controller) enqueueController() {
+	cfg, err := ctrl.ccLister.Get(ctrlcommon.ControllerConfigName)
+	if err != nil {
+		utilruntime.HandleError(fmt.Errorf("couldn't get ControllerConfig on secret callback %#v", err))
+		return
+	}
+	glog.V(4).Infof("Re-syncing ControllerConfig %s due to secret change", cfg.Name)
```

Comment:
This log line is no longer correct if this is being called from `updateFeature` for example, does this need to be more generic?
```suggestion
	glog.V(4).Infof("Re-syncing ControllerConfig %s due to dependency change", cfg.Name)
```